warehouse/admin/templates/admin/banners/edit.html

```diff
@@ -105,8 +105,12 @@ <h3 class="box-title">Create banner</h3>
             {% endif %}
 
             {{ render_field("Name", form.name, "banner-name", class="form-control", placeholder="Name") }}
+
+            {{ render_field("fa_icon", form.fa_icon, "banner-fa-icon", class="form-control", placeholder="Fontawesome class") }}
```

Comment:
Maybe "Icon name" would be clearer as a text field.
Also, a help text with a link (targer=_blank) pointing to the fontawesome icon selector would definitely help.

Reply:
(would it make sense adding a ~1 liner JS script that would display the icon when the field is edited ? I believe it's only worth it if it's painless to write and I don't remember if CSP allow inline JS, but if it doesn't it's definitely not worth it)

Reply:
You're correct, CSP prevents from inline JS in the HTML. Anyway, I think the preview option will be enough for moderator and admin users to make sure they're using a valid class icon.

Reply:
The link to fontawesome selector was added by 7b74938126e7af575361bac53dea5c1b4f72e506